### PR TITLE
[Doc] Add Ingest CSV Processor Plugin to plugin as a community plugin

### PR DESCRIPTION
**Pull request by johtani**
Add my Ingest plugin to doc

### CODE REVIEW

File: docs/plugins/ingest.asciidoc [hunk 1]
```diff
@@ -25,6 +25,13 @@ under the CCA-ShareAlike 3.0 license. For more details see, http://dev.maxmind.c
 
 A processor that extracts details from the User-Agent header value.
 
+[float]
+=== Community contributed repository plugins
```
Comment (cbuescher) [inline docs/plugins/ingest.asciidoc]: I think this should be "ingest plugins" in this case

**Review by cbuescher — CHANGES_REQUESTED** [review]
I left a small comment

File: docs/plugins/ingest.asciidoc (hunk 1 above)
Comment (johtani) [reply docs/plugins/ingest.asciidoc]: Ah, good catch...

**Comment by johtani** [comment]
@cbuescher Thanks for reviewing... I still have jet lag...
Fix your comment.

**Review by cbuescher — APPROVED** [review]
LGTM

**Comment by johtani** [comment]
@elasticmachine test this please

**Pull request merged**